public/electron.js

```diff
@@ -66,7 +64,10 @@ function getOpenedWindow(url,options) {
       fullscreen: false,
       show: false,
     });
-
+    win.webContents.on('new-window', function(event, url){
```

Comment:
should it be unsubscribed at some point?

Reply:
I think it is not necessary since the main window is destroyed when the app is closed.